v2f/src/it/scala/org/broadinstitute/monster/etl/v2f/V2FIntegrationSpec.scala

```diff
@@ -80,7 +86,8 @@ class V2FIntegrationSpec extends PipelineSpec with Matchers with BeforeAndAfterA
       "variant-effect/transcript-consequences/*.json",
       "have written the correct transcript-consequences data"
     ),
-    ("variants/*.json", "have written the correct variants data")
+    ("variants/*.json", "have written the correct variants data"),
+    ("dataset-specific/*/part-*", "have written the correct dataset specific data")
```

Comment:
Did this glob pattern turn out to actually work?

Reply:
Whoa whattt. Maybe the globbing function used here is different? 😨